### PR TITLE
Create ur.json added Urdu

### PR DESCRIPTION
**Pull request by yousaf465**
added Urdu translations

### CODE REVIEW

File: custom_components/ble_monitor/translations/ur.json [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        }
+      },
+      "add_remove_device": {
+        "title": "آلہ شامل کریں",
+        "data": {
+          "mac": "میک ایڈریس",
+          "uuid": "بیکن UUID",
+          "encryption_key": "خفیہ کاری کلید",
+          "temperature_unit": "درجہ حرارت کی اکائی [جیسا کہ آلے کے ذریعہ بھیجی گئی ہے]",
+          "use_median": "Use median instead of mean",
```
Comment (Ernst79) [inline custom_components/ble_monitor/translations/ur.json:27]: Shouldn’t this line be translated ?

**Review by Ernst79 — CHANGES_REQUESTED** [review]
Thanks, one comment, you missed one line I think, can you translate that as well?

**Review by Ernst79 — APPROVED** [review]
Thanks!

**Comment by yousaf465** [comment]
آپ کا بے حد شکریہ !!!

**Pull request merged**